library/src/main/java/com/google/android/exoplayer/text/SubtitleParserHelper.java

```diff
@@ -179,8 +179,13 @@ private void handleSample(long sampleTimeUs, SampleHolder holder) {
       if (sampleHolder != holder) {
         // A flush has occurred since this holder was posted. Do nothing.
       } else {
-        this.result = new PlayableSubtitle(parsedSubtitle, subtitlesAreRelative, sampleTimeUs,
-            subtitleOffsetUs);
+        // it's more accurate to read startTime from eventTime compare to sampleTime
```

Comment:
Gotcha, let's revisit live subtitles on different PR. I'll revert the class for the time being